dockerfile

```diff
@@ -0,0 +1,22 @@
+#Deriving the latest base image
+FROM node:latest
```

Comment:
Takes awhile to download full `node`, might be worth trying to use `node:alpine` or `slim`.. would make initial bootup faster & take less space